data/generalhw_scripts/rpi_pico_w_keyboard/rpi_pico_w_keyboard.ino

```diff
@@ -0,0 +1,200 @@
+#include <WiFi.h>
+#include <WiFiClient.h>
+#include <WebServer.h>
+
+#include <Keyboard.h>
+#include "keymap.h"
+
+
+#ifndef STASSID
+#define STASSID "openQA-worker"
+#define STAPSK "CHANGE THIS DUMMY TO YOUR PASSWORD"
+#endif
+
+const char *ssid = STASSID;
+const char *password = STAPSK;
+WebServer server(80);
+
+
+// Ringbuffer for the last 10 keyboard inputs to be shown on the webui
+#define LOGBUF_LEN 10
+char* logbuf[LOGBUF_LEN] = {NULL};
+uint8_t logbuf_next_idx = 0;
```

Comment:
I recommend to use `uint_fast8_t` unless you need exactly 8 bit. Same recommendation goes for other uses of bit-fixed types.